### PR TITLE
Updates to the wallet

### PR DESCRIPTION
**Pull request by michaelfig**
* Minor cleanup to agoric-cli deploy command.
* Change UI to work properly under the released SES/lockdown that doesn't have a Babel dependency
* Less garish marking of current page (needed to adjust the Smelte Button to react to `{color}`)
* Prune all the react yarn.lock dependencies

### CODE REVIEW

File: packages/dapp-svelte-wallet/ui/package.json [hunk 1]
```diff
@@ -12,13 +12,14 @@
...
     "@agoric/captp": "^1.3.3",
-    "@agoric/harden": "^0.0.6",
     "@rollup/plugin-commonjs": "^12.0.0",
     "@rollup/plugin-node-resolve": "^8.0.0",
+    "livereload-js": "file:livereload-js-v3.3.1.tgz",
     "rollup": "^2.3.4",
     "rollup-plugin-livereload": "^1.0.0",
     "rollup-plugin-svelte": "^5.0.3",
     "rollup-plugin-terser": "^6.1.0",
+    "ses": "^0.10.0",
```
Comment (dtribble) [inline packages/dapp-svelte-wallet/ui/package.json:22]: is it correct for this to be a devDependency?

**Review by dtribble — COMMENTED** [review]
LGTM

File: packages/dapp-svelte-wallet/ui/package.json (hunk 1 above)
Comment (warner) [reply packages/dapp-svelte-wallet/ui/package.json:22]: I suspect not, most packages import `install-ses` if anything. I'll take a closer look.

Comment (warner) [reply packages/dapp-svelte-wallet/ui/package.json:22]: ah, ok, this package defines it's own version of `@agoric/install-ses` (with more taming than `@agoric/install-ses` uses), so yes it needs `ses` as a direct dependency

Comment (warner) [reply packages/dapp-svelte-wallet/ui/package.json:22]: also it uses the new `ses/lockdown` subset, which is less than what `@agoric/install-ses` provides (`Compartment`)

Comment (michaelfig) [reply packages/dapp-svelte-wallet/ui/package.json:22]: It's correct.  We import `ses/lockdown` directly to avoid taking a babel dependency.

**Review by warner — APPROVED** [review]
Is it really necessary to include a 93kB binary tarball (`livereload-js-v3.3.1.tgz`) in the source tree? The reduction of `yarn.lock` dependencies more than makes up for it, but it still feels weird.

The SES stuff looks good, nice use of the new smaller export.

**Comment by michaelfig** [comment]
> Is it really necessary to include a 93kB binary tarball (`livereload-js-v3.3.1.tgz`) in the source tree? The reduction of `yarn.lock` dependencies more than makes up for it, but it still feels weird.

I felt uncomfortable doing it, so thanks for the question. 🙂 

I just now learned about [URL-based Git packages in Yarn](https://classic.yarnpkg.com/en/docs/cli/add#toc-adding-dependencies), so I've replaced the .tgz with that instead.  Much better! 🚀

**Pull request merged**